### PR TITLE
hovercraft: 2.0 -> 2.6

### PR DESCRIPTION
**Pull request by makefu**
###### Motivation for this change

move to top-level, use buildPythonApplication, bump version to 2.6 .
hovercraft tool for creating presentations, not a python library.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by makefu** [comment]
even the tests run through on aarch64 and darwin!

File: pkgs/applications/misc/hovercraft/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+, pygments
+, watchdog
+}:
+
+buildPythonApplication rec {
+  pname = "hovercraft";
+  version = "2.6";
+  disabled = ! isPy3k;
+
+  src = fetchFromGitHub {
```
Comment (worldofpeace) [inline pkgs/applications/misc/hovercraft/default.nix]: I'm assuming here that the tests weren't distributed at `pypi`?

File: pkgs/applications/misc/hovercraft/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  };
+
+  checkInputs = [ manuel ];
+  propagatedBuildInputs = [ setuptools docutils lxml svg-path pygments watchdog ];
+
+  meta = with lib; {
+    description = "Makes impress.js presentations from reStructuredText";
+    homepage = https://github.com/regebro/hovercraft;
+    license = licenses.mit;
+    maintainers = with maintainers;[ goibhniu makefu ];
```
Comment (worldofpeace) [inline pkgs/applications/misc/hovercraft/default.nix]:
```suggestion
    maintainers = with maintainers; [ goibhniu makefu ];
```

File: pkgs/applications/misc/hovercraft/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/applications/misc/hovercraft/default.nix]: yes: https://github.com/regebro/hovercraft/blob/master/setup.py#L42

Comment (makefu) [reply pkgs/applications/misc/hovercraft/default.nix]: exactly, i was using pypi before

**Pull request merged**